### DISCUSSION TITLE
Welcome to autophot Discussions!

### DISCUSSION THREAD

**Discussion by Astro-Sean** [Astro-Sean/autophot#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yanshengyu0311** [comment]
Hi, I am Shengyu Yan, a Ph.D. student of Prof. Xiaofeng Wang, Tsinghua University. Yongzhi Cai suggest I use your pipeline to reduced the photometric image. I have read parts of your code, I can learn lots of reduction experience from your code.
